### PR TITLE
fix(user): supply a kubectl patch function to simplify registration mode setting

### PR DESCRIPTION
**Pull request by helgi**

### CODE REVIEW

File: src/users/registration.md [hunk 1]
```diff
@@ -62,11 +62,11 @@ Deis Workflow supports three registration modes:
 | admin\_only       | Only existing admins may register new users     |
 
 To modify the registration mode for Workflow you may add or modify the `REGISTRATION_MODE` environment variable for the
-controller component. If Deis Workflow is already running, use `kubectl --namespace=deis edit deployment deis-controller`:
+controller component. If Deis Workflow is already running, use:
 
-Find the `REGISTRATION_MODE` environment variable in the template file or add the appropriate section:
+`kubectl patch deployments deis-controller -p '{"spec":{"template":{"spec":{"containers":[{"name":"deis-controller","env":[{"name":"REGISTRATION_MODE","value":"disabled"}]}]}}}}'`
 
-![](../images/controller-registration-mode.png)
```
Comment (slack) [inline src/users/registration.md]: Also remove the png from the repo.

File: src/users/registration.md [hunk 2]
```diff
@@ -62,11 +62,11 @@ Deis Workflow supports three registration modes:
...
+Modify the `value` portion to match the one of the desired modes.
```
Comment (slack) [inline src/users/registration.md]: "to match the desired mode."

File: src/users/registration.md (hunk 1 above)
Comment (helgi) [inline src/users/registration.md]: ? It's already gone...

Comment (slack) [inline src/users/registration.md]: The file?

Comment (helgi) [inline src/users/registration.md]: you

**Pull request merged**